view/SSTemplateParser.php

```diff
@@ -3775,6 +3780,8 @@ function ClosedBlock_Handle_With(&$res) {
 		}
 		
 		$on = str_replace('$$FINAL', 'obj', ($arg['ArgumentMode'] == 'default') ? $arg['lookup_php'] : $arg['php']);
+		// obj() has a fourth argument to enable caching, so add that to the end of the call
+		$on = preg_replace('/(\))$/', ', true)', $on);
```

Comment:
This looks like risky code. You're using a regexp to hack in an additional parameter to a method call?